### PR TITLE
Add created_time for ASG

### PR DESCRIPTION
**Pull request by oporkka**
Added created_time for auto scaling groups in order to allow detection
of old obsolete stack versions.

Resolves https://github.com/zalando-zmon/zmon-aws-agent/issues/139

### CODE REVIEW

File: zmon_aws_agent/aws.py [hunk 1]
```diff
@@ -507,6 +507,7 @@ def get_auto_scaling_groups(region, acc, **kwargs):
             'desired_capacity': g['DesiredCapacity'],
             'max_size': g['MaxSize'],
             'min_size': g['MinSize'],
+            'created_time': str(g['CreatedTime']),
```
Comment (oporkka) [inline zmon_aws_agent/aws.py]: I'm not sure if `str` is redundant here, because this will be later converted to JSON.

File: tests/conftest.py [hunk 2]
```diff
@@ -78,7 +79,8 @@ def get_autoscaling():
             {'InstanceId': 'ins-2', 'LifecycleState': 'InService'},
             {'InstanceId': 'ins-3', 'LifecycleState': 'InService'},
             {'InstanceId': 'ins-4', 'LifecycleState': 'unknown'},
-        ]
+        ],
+        'CreatedTime': datetime(2018, 6, 6, 9, 59, 38, 127000, tzinfo=tzutc())
```
Comment (oporkka) [inline tests/conftest.py]: This is how it is returned from https://github.com/zalando-zmon/zmon-aws-agent/blob/b2877e63a02b92e3e64d76cef6dd22b2839c1ed5/zmon_aws_agent/aws.py#L491-L496

File: tests/conftest.py [hunk 3]
```diff
@@ -111,6 +113,7 @@ def get_autoscaling():
             'max_size': 10,
             'min_size': 3,
             'instances': [{'aws_id': 'ins-1', 'ip': '192.168.20.16'}],
+            'created_time': '2018-06-06 09:59:38.127000+00:00'
```
Comment (oporkka) [inline tests/conftest.py]: Does ZMON have some standard formatting? `last_modified` has format of type `"2017-08-11 11:35:15.170484"` but I'm not sure which part of ZMON infrastructure adds that to the type.

Comment (oporkka) [reply tests/conftest.py]: Could be this one https://github.com/zalando-zmon/zmon-cli/blob/3caa3d97632079feb4fb4ab0c41dd25f864dcfd8/zmon_cli/cmds/entity.py#L19-L23

Comment (hjacobs) [reply tests/conftest.py]: @oporkka the format should probably be ISO format like `YYYY-MM-DD HH:MM:SS` which can be parsed via `time(...)`, see https://docs.zmon.io/en/latest/user/check-commands.html#time-helper-function

Comment (oporkka) [reply tests/conftest.py]: Thanks, changed! Basicly it meant leaving out the timezone information `'2018-06-06 09:59:38.127000+00:00'` ->  `'2018-06-06 09:59:38.127000'`

**Comment by mohabusama** [comment]
👍

**Comment by mohabusama** [comment]
Thanks @oporkka

**Pull request merged**